source/form.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="ru">
+
+  <head>
+    <meta charset="utf-8">
+    <title>HTML Academy: Кэт энерджи</title>
+  </head>
+
+  <body>
+    <header class="header">
+      <a class="header__logo" href="">
+        <img class="header__logo-image" src="https://via.placeholder.com/191x39" width="191" height="39"
+          alt="Логотип Кэт энерджи">
+      </a>
+      <nav class="nav">
+        <button class="nav__toggle">Открыть меню</button>
+        <div class="nav__wrapper">
+          <ul class="nav__menu site-menu">
+            <li class="site-menu__item site-menu__item--active">
+              <a>Главная</a>
+            </li>
+            <li class="site-menu__item">
+              <a href="#">Каталог продукции</a>
+            </li>
+            <li class="site-menu__item">
+              <a href="#">Подбор программы</a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <footer class="footer">
+      <a class="footer__logo" href="index.html">
+        <img class="footer__logo-image" src="https://via.placeholder.com/128x24" width="128" height="24"
+          alt="Логотип Кэт энерджи">
+      </a>
+      <div class="social footer__social">
```

Comment:
```suggestion
      <div class="footer__social social">
```

Мелочь, а так будет читабельней.